app/assets/javascripts/dashboard/views/dash/explorative_annotations_view.js

```diff
@@ -360,6 +461,7 @@ export default class ExplorativeAnnotationsView extends React.PureComponent {
               {search}
             </div>}
         <h3>Explorative Annotations</h3>
+        {this.renderSearchTags()}
```

Comment:
Do you think it's clear to everyone how the tag search works? Maybe we should add a label like "Current tag filters" or a tooltip?